src/set.js

```diff
@@ -0,0 +1,94 @@
+export default class Set {
+  constructor() {
+    this.length = 0
+    this.elements = []
+  }
+
+  add(element) {
+    this.length === 0
```

Comment:
I can't figure out why this ternary is used. It seems that the same operation would occur if you used push or the index to place the element in the array.